data-prepper-core/src/main/java/org/opensearch/dataprepper/peerforwarder/PeerForwarderConfiguration.java

```diff
@@ -85,6 +86,7 @@ public PeerForwarderConfiguration (
             @JsonProperty("domain_name") final String domainName,
             @JsonProperty("static_endpoints") final List<String> staticEndpoints,
             @JsonProperty("client_thread_count") final Integer clientThreadCount,
+            @JsonProperty("client_thread_pool_bound") final Integer clientThreadPoolBound,
```

Comment:
I wonder if there is a clearer name here. This is defining how many requests can be queued up without blocking, right?

Perhaps: `client_worker_queue_depth`, `client_outgoing_queue_depth`?

Thoughts?